.github/workflows/auto-deploy-staging.yml

```diff
@@ -0,0 +1,68 @@
+name: Auto deploy staging AWS
+env: 
+  NODE_VERSION: '12.x'
```

Comment:
I keep forgetting, why are we stuck on node 12?  gh-actions workers have node 14 something auto installed

Reply:
It's the version we were using when we started the cfe repo I think, we can update it. For a while we had both 12 and 14. I think it's pretty standard to include 2 so you can feel comfortable going up to the next version.